packages/messenger-widget/src/components/ConfigureProfile/bl.tsx

```diff
@@ -213,3 +211,48 @@ export const fetchExistingDM3Name = async (
         setExistingDm3Name(null);
     }
 };
+
+const enum NAME_SERVICES {
+    ENS = 'Ethereum Network - Ethereum Name Service (ENS)',
+    GENOME = 'Gnosis Network - Genome/SpaceID',
+}
+
+export const namingServices = [
+    {
+        name: NAME_SERVICES.ENS,
+        chainId: [1],
+        component: <ConfigureEnsProfile />,
+    },
+    {
+        name: NAME_SERVICES.GENOME,
+        chainId: [100],
```

Comment:
Why is chainId an array and not a number ?

Reply:
Earlier there were 2 chainIds used for Gnosis, so I thought might be possible can have multiple chain ID's kept as an array

Reply:
that was gnosis chain and gnosis-test. Those are independent chains with separate name services. Please change the cainId param to number

Reply:
ok